<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="windowTitle">
   <string>单词测试</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="questionLabel">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="option1"/>
   </item>
   <item>
    <widget class="QRadioButton" name="option2"/>
   </item>
   <item>
    <widget class="QRadioButton" name="option3"/>
   </item>
   <item>
    <widget class="QRadioButton" name="option4"/>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>